build-logic-commons/basics/src/main/kotlin/gradlebuild/basics/tasks/PackageListGenerator.kt

```diff
@@ -72,6 +73,11 @@ abstract class PackageListGenerator : DefaultTask() {
             "sun/misc"
         )
 
+        // Things we do want to shade despite being covered by excludes
+        val DEFAULT_INCLUDES: List<String> = listOf(
+            "org/gradle/fileevents"
+        )
```

Comment:
💭 The changes in this class are about forcing the external `fileevents` library to be shaded like the other external libraries. The explicit inclusion of `org/gradle/fileevents` is required because we exclude `org/gradle` by default. Historically, `org/gradle` was only used for code originating from the Gradle build tool, and thus not needed to be shaded. But `gradle-fileevents` now introduces an external library that is under `org/gradle`, so we need this special treatment.